ts/@live-compositor/node/src/fetch.ts

```diff
@@ -33,6 +34,28 @@ export async function sendRequest(baseUrl: string, request: ApiRequest): Promise
   return (await response.json()) as object;
 }
 
+export async function sendMultipartRequest(
+  baseUrl: string,
+  request: MultipartRequest
+): Promise<object> {
+  const response = await fetch(new URL(request.route, baseUrl), {
+    method: request.method,
+    body: request.body as FormData,
+    agent: url => (url.protocol === 'http:' ? httpAgent : httpsAgent),
+  });
+  if (response.status >= 400) {
+    const err: any = new Error(`Multipart request to compositor failed.`);
```

Comment:
```suggestion
    const err: any = new Error(`Request to compositor failed.`);
```
multipart is not important information for the user. If you want to add more information it should be maybe route/error code/response body, but whether it is multipart or json should not mater to someone that needs to handle this exception